.github/workflows/zitadel.yml

```diff
@@ -78,3 +78,11 @@ jobs:
         with:
           file: .artifacts/codecov/profile.cov
           name: go-codecov
+      - name: Bump Chart Version
+        uses: peter-evans/repository-dispatch@v2
+        if: steps.semantic.outputs.new_release_published == 'true' && github.ref == 'refs/heads/main'
+        with:
+          token: ${{ secrets.CHART_REPO_ACCESS_TOKEN }}
```

Comment:
Is this a pat of you or the same GH App we use for homebrew/zitadel-go?

Reply:
It's not yet added to the repo.

Reply:
https://github.com/zitadel/zitadel/blob/3dee8525d892c10463c8d6ab3328f0f69f297c7c/.github/workflows/release-channels.yml#L38

Reply:
Permission wise I can change the access of the GH App

Reply:
Yes you can use it there and it has access to the following repos
![image](https://user-images.githubusercontent.com/9879976/195558894-fb5c10c0-9bdf-4d2b-9d6f-acafbe0061b4.png)

Reply:
Perfect. Can you add the required secrets to the charts repo, please? Or do I have access to the values?

Reply:
@fforootd is that done and can we merge it?